mrmustard/physics/triples.py

```diff
@@ -218,6 +218,33 @@ def two_mode_squeezed_vacuum_state_Abc(
     return A, b, c
 
 
+def quadrature_eigenstates_Abc(x: float, phi: float) -> Union[Matrix, Vector, Scalar]:
```

Comment:
Not something for this PR and not anything major but for the future I would prefer we move away from `Union[...]` for type hints and instead use `|` (e.g. `Matrix | Vector | Scalar`). I just find it concise and it avoids the need to import Union. We could also replace `Optional[...]` with `... | None`. Tagging @ziofil for visibility

Reply:
I agree